static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -223,54 +223,52 @@ function SummaryContent({
-      spanOperationBreakdownFilter,
-      location
-    );
+  const spanOperationBreakdownConditions = filterToSearchConditions(
+    spanOperationBreakdownFilter,
+    location
+  );
 
-    if (spanOperationBreakdownConditions) {
-      eventView = eventView.clone();
-      eventView.query = `${eventView.query} ${spanOperationBreakdownConditions}`.trim();
-      transactionsListEventView = eventView.clone();
-    }
+  if (spanOperationBreakdownConditions) {
+    eventView = eventView.clone();
+    eventView.query = `${eventView.query} ${spanOperationBreakdownConditions}`.trim();
+    transactionsListEventView = eventView.clone();
+  }
 
-    // update header titles of transactions list
+  // update header titles of transactions list
 
-    const operationDurationTableTitle =
-      spanOperationBreakdownFilter === SpanOperationBreakdownFilter.None
-        ? t('operation duration')
-        : `${spanOperationBreakdownFilter} duration`;
+  const operationDurationTableTitle =
+    spanOperationBreakdownFilter === SpanOperationBreakdownFilter.None
+      ? t('operation duration')
+      : `${spanOperationBreakdownFilter} duration`;
 
-    // add ops breakdown duration column as the 3rd column
-    transactionsListTitles.splice(2, 0, operationDurationTableTitle);
+  // add ops breakdown duration column as the 3rd column
+  transactionsListTitles.splice(2, 0, operationDurationTableTitle);
 
-    // span_ops_breakdown.relative is a preserved name and a marker for the associated
-    // field renderer to be used to generate the relative ops breakdown
-    let durationField = SPAN_OP_RELATIVE_BREAKDOWN_FIELD;
+  // span_ops_breakdown.relative is a preserved name and a marker for the associated
+  // field renderer to be used to generate the relative ops breakdown
+  let durationField = SPAN_OP_RELATIVE_BREAKDOWN_FIELD;
 
-    if (spanOperationBreakdownFilter !== SpanOperationBreakdownFilter.None) {
-      durationField = filterToField(spanOperationBreakdownFilter)!;
-    }
+  if (spanOperationBreakdownFilter !== SpanOperationBreakdownFilter.None) {
+    durationField = filterToField(spanOperationBreakdownFilter)!;
+  }
 
-    const fields = [...transactionsListEventView.fields];
+  const fields = [...transactionsListEventView.fields];
 
-    // add ops breakdown duration column as the 3rd column
-    fields.splice(2, 0, {field: durationField});
+  // add ops breakdown duration column as the 3rd column
+  fields.splice(2, 0, {field: durationField});
 
-    if (spanOperationBreakdownFilter === SpanOperationBreakdownFilter.None) {
-      fields.push(
-        ...SPAN_OP_BREAKDOWN_FIELDS.map(field => {
-          return {field};
-        })
-      );
-    }
-
-    transactionsListEventView.fields = fields;
+  if (spanOperationBreakdownFilter === SpanOperationBreakdownFilter.None) {
+    fields.push(
+      ...SPAN_OP_BREAKDOWN_FIELDS.map(field => {
+        return {field};
+      })
+    );
   }
```

Comment:
🤔 Wondering how these weren't formatted, I thought the precommit covers that.

Reply:
I think it's because all of this code was wrapped in an if statement which I removed, and everything within was un-indented

Reply:
Ohh yeah lol, you're right. In that case it was formatted fine!